bin/roger.py

```diff
@@ -21,6 +21,7 @@ def roger_version(root_dir):
                 version = f.read().strip()
     finally:
         print(version)
+    return version
```

Comment:
The `print(version)` line will end up printing the version all the time. Print it only when `-v` or `--version` is specified.